src/zc/buildout/easy_install.py

```diff
@@ -513,6 +516,9 @@ def _get_dist(self, requirement, ws, for_buildout_run=False):
                     raise zc.buildout.UserError(
                         "Couldn't download distribution %s." % avail)
 
+                if dist.location.endswith('.whl'):
+                    dist = wheel_to_egg(dist, tmp)
```

Comment:
In case the download cache is activated, `tmp` here **is** the download cache.

The result is that the download cache now has the egg AND the wheel at the same time. And the egg gets overwritten with the contents of the wheel every time it's written.

Humpty will gladly do it for you, so you'll never notice the constant overwriting, but it bit me trying to replace humpty with `wheel install` with an exception like:

    ValueError: Wheel file zest/releaser/tests/gitsvn.txt would overwrite /path/to/buildout/downloads/dist/zest.releaser-6.9-py2.7.egg/zest/releaser/tests/gitsvn.txt. Use force if this is intended

An actual, separate, temporary directory needs to be used in this case.